modules/cudaarithm/perf/perf_element_operations.cpp

```diff
@@ -1423,17 +1424,19 @@ PERF_TEST_P(Sz_AngleInDegrees, CartToPolar,
 //////////////////////////////////////////////////////////////////////
 // PolarToCart
 
-PERF_TEST_P(Sz_AngleInDegrees, PolarToCart,
+PERF_TEST_P(Sz_Type_AngleInDegrees, PolarToCart,
             Combine(CUDA_TYPICAL_MAT_SIZES,
+                    testing::Values(CV_32FC1, CV_64FC1),
                     Bool()))
```

Comment:
There is `CUDA_SANITY_CHECK()` below.
It is very sensitive to test name: run perf test binary with "--perf_verify_sanity".
So, need to update sanity data too (opencv_exta/testdata/perf) or "disable" these checks (like `SANITY_CHECK_NOTHING()`).

Reply:
I made a PR against extras repo.
BTW: I noticed that the single-precision version is not very consistent with its results. This have nothing to do with proposed PR though. It seems we might need to relax the perf tolerance for it as well.